apps/platform_runner/test/end_to_end_test.exs

```diff
@@ -479,4 +479,13 @@ defmodule PlatformRunner.EndToEndTest do
       end
     end
   end
+
+  describe "metrics" do
+    test "exposes metrics for all services" do
+      for port <- [9569, 9570, 9572, 9573, 9574, 9576] do
```

Comment:
i assigned 9571 and 9575 to `define` and `profile` respectively going through the list and i'm pretty sure they're not currently running during the e2e test. When they are, this list should just be converted to a range object (`9569..9576`)